api/envoy/api/v2/core/protocol.proto

```diff
@@ -32,6 +32,12 @@ message UpstreamHttpProtocolOptions {
   // :ref:`router filter <config_http_filters_router>`.
   // This field is intended to set with `auto_sni` field.
   bool auto_san_validation = 2;
```

Comment:
We need to describe this additional behavior on `auto_san_validation`. What we should add is that the presented value for SAN validation will be changed with `alt_header_name`.

Reply:
Sure, will add the needed description.